packages/pg-v5/test/unit/lib/host.unit.test.js

```diff
@@ -5,40 +5,23 @@ const {expect} = require('chai')
 let host = require('../../../lib/host')
 
 describe('host', () => {
-  it('shows dev host', () => {
-    expect(host({plan: {name: 'heroku-postgresql:mini'}})).to.equal('https://postgres-starter-api.heroku.com')
-  })
-
-  it('shows prod host', () => {
-    expect(host({plan: {name: 'heroku-postgresql:premium-0'}})).to.equal('https://postgres-api.heroku.com')
-  })
-
-  context('for numbered essential plans', () => {
-    it('shows data host', () => {
-      expect(host({plan: {name: 'heroku-postgresql:essential-0'}})).to.equal('https://postgres-api.heroku.com')
-    })
+  it('shows api.data.heroku.com host', () => {
+    expect(host({plan: {name: 'heroku-postgresql:mini'}})).to.equal('https://api.data.heroku.com')
+    expect(host({plan: {name: 'heroku-postgresql:premium-0'}})).to.equal('https://api.data.heroku.com')
+    expect(host({plan: {name: 'heroku-postgresql:essential-0'}})).to.equal('https://api.data.heroku.com')
   })
 
   context('with HEROKU_DATA_HOST set', () => {
     beforeEach(() => {
       process.env.HEROKU_DATA_HOST = 'data-host.herokuapp.com'
-      process.env.HEROKU_POSTGRESQL_HOST = 'postgresql-host.herokuapp.com'
     })
     afterEach(() => delete process.env.HEROKU_DATA_HOST)
     afterEach(() => delete process.env.HEROKU_POSTGRESQL_HOST)
```

Comment:
Is this still required even when you removed setting this env var on line 25?

Reply:
Nope, removed.